### PR TITLE
New User Email Confirmation - emailjs

### PR DESCRIPTION
**Pull request by danielsarap96**
- New user confirmation email using emailjs

### CODE REVIEW

File: client/src/app/util/EmailVerification.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import emailjs, {init} from '@emailjs/browser';
+init('5A9OSqglcYcpKAMgd');
+
+export const VerifyEmail = (user) => {
+  const templateParams = {
+    to_email: user.useremail,
+    message: `http://localhost:3000/verify/${user.token}`,
```
Comment (colinaguan) [inline client/src/app/util/EmailVerification.js]: could you either add a comment here to remind us to make it dynamic, or edit it and merge?

Comment (danielsarap96) [reply client/src/app/util/EmailVerification.js]: I added a comment to the pr for now while I figure out how to make the dynamic link work.

Comment (danielsarap96) [reply client/src/app/util/EmailVerification.js]: The link is now dynamic

**Review by zkm1 — APPROVED** [review]
funtionally working!

File: client/src/app/pages/Login.js [hunk 2]
```diff
@@ -69,6 +67,13 @@ export default function Login() {
     }
   }, [user]);
 
+
+  const handleResendClick = () => {
+    console.log('here');
+    console.log(createdProfileData);
+    <>{verifyEmail(createdProfileData)}</>;
```
Comment (colinaguan) [inline client/src/app/pages/Login.js]: is there not a way to do this without putting it the html component brackets?

File: client/src/app/pages/Login.js [hunk 3]
```diff
@@ -101,9 +105,14 @@ export default function Login() {
             draggable: true,
             progress: undefined,
           });
-          setUser(json);
-          setLoggedIn(true);
-          navigate(`/`);
+          setCreatedProfileData(json);
+          <>{verifyEmail(json)}</>;
```
Comment (colinaguan) [inline client/src/app/pages/Login.js]: (same as above)

File: client/src/app/pages/Login.js (hunk 2 above)
Comment (danielsarap96) [reply client/src/app/pages/Login.js]: Not that I could figure out, React is pretty stupid in that there is no easy way to call functional components as if they are functions. If anyone knows a way to call it normally that would be great.

**Pull request merged**